proposed/clock-meta.md

```diff
@@ -0,0 +1,80 @@
+# PHPDoc Meta Document
+
+## 1. Summary
+
+The purpose of using the ClockInterface would allow mocking time in many situations where
```

Comment:
Not sure if "mocking time" is the purpose.

I think it is worth mentioning @martinfowler's article here: https://martinfowler.com/bliki/ClockWrapper.html.

Reply:
Thanks, i have added it to the links